packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -128,7 +128,7 @@ const getReactRouteGroups = (showReactApp, reactRoute) => {
         'post_verify/third_party_auth/callback',
         'post_verify/third_party_auth/set_password',
       ]),
-      fullProdRollout: false,
+      fullProdRollout: true,
```

Comment:
This won't roll out until we also turn the feature flag on. I'll double check that's set up properly.

Reply:
PR on SRE side: https://github.com/mozilla-it/webservices-infra/pull/3382